src/main/java/ch/naviqore/raptor/Connection.java

```diff
@@ -1,21 +1,22 @@
 package ch.naviqore.raptor;
 
+import java.time.LocalDateTime;
 import java.util.List;
 
 /**
  * A connection is a sequence of legs to travel from an origin stop to destination stop.
  */
 public interface Connection {
 
-    int getDepartureTime();
+    LocalDateTime getDepartureTime();
 
-    int getArrivalTime();
+    LocalDateTime getArrivalTime();
 
     String getFromStopId();
 
     String getToStopId();
 
-    int getDuration();
+    int getDurationInSeconds();
```

Comment:
Maybe return a `Duration`? https://docs.oracle.com/javase/8/docs/api/java/time/Duration.html

Reply:
Was thinking of the same but didn't want to over engineer this for now.

Reply:
We could also omit this field completely, since it is kind of redundant?